src/include/sof/audio/buffer.h

```diff
@@ -8,6 +8,7 @@
 #ifndef __SOF_AUDIO_BUFFER_H__
 #define __SOF_AUDIO_BUFFER_H__
 
+#include <sof/audio/audio_stream.h>
```

Comment:
What with the todo in the subjec?

Reply:
This commit is quite large and is under development